README.md

```diff
@@ -343,6 +343,92 @@ $ plextraktsync --server=Example1 info
 INFO     Enabled 2 libraries in Plex Server: ['Movies', 'TV Shows']
 ```
 
+### Per server configuration
+
+If you want to specify your config per server you can do so inside of `servers.yml`. Within the `config` part of the server configuration you can specify how that specific server should work.
+
+```yml
+  Example1:
+    token: ~
+    urls:
+      - http://localhost:32400
+    config:
+      sync:
+        plex_to_trakt:
+          collection: true
+        trakt_to_plex:
+          liked_lists: false
+```
+Using `sync` in a server config overrides the global sync-config in `config.yml`.
+
+This can also be used to have different configs between different libraries. To be able to do this you specifiy the number of servers you need (most likely equal to the number of different config setups you need). For example:
+
+```yml
+  Example1:
+    token: ~
+    urls:
+      - http://localhost:32400
+    config:
+      libraries:
+        - "Movies"
+      sync:
+        plex_to_trakt:
+          ratings: true
+          watched_status: true
+        trakt_to_plex:
+          ratings: true
+          watched_status: true
+  Example2:
+    token: ~
+    urls:
+      - http://localhost:32400
+    config:
+      libraries:
+        - "TV Shows"
+      sync:
+        plex_to_trakt:
+          ratings: true
+          watched_status: false
+        trakt_to_plex:
+          ratings: true
+          watched_status: false
+```
+The above config would make it so that the "Movies" library syncs both ratings and watched status, while the "TV Shows" library only syncs ratings.
+To then run the sync you need to specify `--server Example1` or `--server Example2` to run the sync for that specific server. Running the sync command without it will result in running the sync using the default config.
```

Comment:
```suggestion
To then run the sync you need to specify `--server Example1` or `--server Example2` to run the sync for that specific server.

Running the sync command without `--server` will use default server from `.env`
```

again. not true statement detected. --server doesn't specify what configs to override, but rather what server to pick. your default server could be either of those examples or something completely different.

Reply:
Yea! sorry.
Didnt know that default server were saved somewhere.